tests/person_spec.rb

```diff
@@ -0,0 +1,38 @@
+require_relative '../person'
+require_relative '../rental'
+
+describe Person do
+  describe '#add_rental' do
+    let(:book) { Book.new('Title', 'Author') }
+    let(:date) { Date.today }
+    let(:person) { Person.new(20, 'John Doe') }
+
+    it 'adds a new rental to the person' do
+      expect do
+        person.add_rental(book, date)
+      end.to change { person.rentals.length }.by(1)
+    end
+
+    it 'creates a rental with correct attributes' do
+      person.add_rental(book, date)
+      rental = person.rentals.first
+
+      expect(rental.date).to eq(date)
+      expect(rental.person).to eq(person)
+      expect(rental.book).to eq(book)
```

Comment:
- [ ] As pointed out earlier, please kindly include a test for the `can_use_services` method.